.buildkite/scripts/steps/code_coverage/ftr_configs.sh

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/scripts/common/util.sh
+source .buildkite/scripts/steps/code_coverage/util.sh
+
+export CODE_COVERAGE=1 # Kibana is bootstrapped differently for code coverage
+echo "--- Print KIBANA_DIR"
+echo "### KIBANA_DIR: $KIBANA_DIR"
+.buildkite/scripts/bootstrap.sh
+
+echo "--- Build Platform Plugins"
+NODE_OPTIONS=--max_old_space_size=14336 node scripts/build_kibana_platform_plugins \
+  --no-examples --test-plugins --workers 4
+
+is_test_execution_step
+
+export JOB_NUM=$BUILDKITE_PARALLEL_JOB
+export JOB=ftr-configs-${JOB_NUM}
+
+FAILED_CONFIGS_KEY="${BUILDKITE_STEP_ID}${BUILDKITE_PARALLEL_JOB:-0}"
+
+# a FTR failure will result in the script returning an exit code of 10
+exitCode=0
+
+configs="${FTR_CONFIG:-}"
+
+if [[ "$configs" == "" ]]; then
+  echo "--- downloading ftr test run order"
+  buildkite-agent artifact download ftr_run_order.json .
+  #  configs=$(jq -r '.groups[env.JOB_NUM | tonumber].names | first' ftr_run_order.json)
+  configs=$(jq -r '.groups[env.JOB_NUM | tonumber].names | .[]' ftr_run_order.json)
```

Comment:
@wayneseymour I think we don't need a commented out one?

Reply:
Indeed, that was for running just one config.  I'll drop it.